session.go

```diff
@@ -94,8 +106,25 @@ func (s *Session) startGC() {
 	}()
 }
 
+// Check if the remote IP is in a trusted network.
+func (s *Session) isRequestFromTrustedNetwork(ctx *fasthttp.RequestCtx) bool {
```

Comment:
I prefer the arguments is a remote ip instead of the complete context.

Reply:
yep, sure.